src/site/content/en/lighthouse-pwa/apple-touch-icon/index.md

```diff
@@ -2,32 +2,31 @@
 layout: post
 title: Does not provide a valid apple-touch-icon
 description: |
-  How to specify what icon should appear on iOS home screens for your Progressive Web App.
-date: 2019-08-26
+  Learn how to specify what icon your Progressive Web App displays on iOS home screens.
 web_lighthouse:
   - apple-touch-icon
 codelabs: codelab-apple-touch-icon
+date: 2019-08-27
+updated: 2019-09-19
 ---
 
-iOS Safari users can manually add [Progressive Web Apps (PWAs)](/discover-installable) to 
-their home screens. The icon that appears on the iOS home screen when the user adds a PWA is
-called the *Apple touch icon*. You can specify what icon your app should use by including a 
+When iOS Safari users add [Progressive Web Apps (PWAs)](/discover-installable) to
+their home screens, the icon that appears is called the *Apple touch icon*.
+You can specify what icon your app should use by including a
 `<link rel="apple-touch-icon" href="/example.png">` tag in the `<head>` of your
 page. If your page doesn't have this link tag, iOS generates an icon by taking a screenshot of
-the page content. In other words, instructing iOS to download an icon results in a more polished 
+the page content. In other words, instructing iOS to download an icon results in a more polished
 user experience.
 
-## How this audit fails
+## How the Lighthouse Apple touch icon audit fails
```

Comment:
I noticed some examples still say "How this audit fails."

- works-offline
- offline-start-url
- redirects-http
- without-javascript

Reply:
That was a compromise because I couldn't think of a clear shorthand for these audits. If @kaycebasques has ideas, happy for the help.

Reply:
I addressed this in my review suggestions. I'm not too worried about the exact phrasing, just wanted to keep Michael's nice heading consistency going.